responsibleai/tests/test_model_analysis_validations.py

```diff
@@ -424,18 +426,20 @@ def test_desired_class_not_set(self):
                 method='random')
 
     def test_desired_range_not_set(self):
-        X_train, y_train, X_test, y_test, _ = \
-            create_binary_classification_dataset()
+        X_train, X_test, y_train, y_test, feature_names = \
+            create_housing_data()
 
-        model = create_lightgbm_classifier(X_train, y_train)
-        X_train['target'] = y_train
-        X_test['target'] = y_test
+        model = create_sklearn_random_forest_regressor(X_train, y_train)
+        X_train = pd.DataFrame(X_train, columns=feature_names)
+        X_test = pd.DataFrame(X_test, columns=feature_names)
+        X_train['TARGET'] = y_train
+        X_test['TARGET'] = y_test
 
         model_analysis = ModelAnalysis(
             model=model,
             train=X_train,
             test=X_test,
-            target_column='target',
+            target_column='TARGET',
```

Comment:
nit: could be a constant...

Reply:
This code is going to be removed once `ModelAnalysis` is deprecated. So refactoring the target string into a variable will give little benefit.